pkg/platform/kube/consumer.go

```diff
@@ -48,6 +50,12 @@ func newConsumer(logger logger.Logger, kubeconfigPath string) (*consumer, error)
 		return nil, errors.Wrap(err, "Failed to create REST config")
 	}
 
+	// add bearer token if specified in environment
+	token := os.Getenv("NUCLIO_BEARER_TOKEN")
```

Comment:
Maybe changing the env name to something less generic?
```suggestion
	token := os.Getenv("NUCLIO_KUBE_CONSUMER_BEARER_TOKEN")
```

Reply:
Thanks, done that now